assets/scss/components/stencil/navUser/_navUser.scss

```diff
@@ -291,7 +291,7 @@
     }
 
     &.is-open {
-        display: initial;
+        display: block;
```

Comment:
Does it affect other browser behaviours?

Reply:
I believe no, because block is init value for div display prop